### DISCUSSION TITLE
Left column of alpinejs docs site doesn't work on iPad safari or chrome

### DISCUSSION THREAD

**Discussion by davidsalazar** [alpinejs/alpine#3500] [7. Bugs (archived)]
Hello, this isn't a bug for alpinejs but for the docs itself and I didn't know where else to put it. Just an fyi. When you look at the docs page on iPad, you will have trouble using the scrolling <aside> left nav bar. It's probably because you have y-overflow auto only available to hover devices and not touch devices. 

Thanks.

**Comment by trych** [comment]
Had the same issue. At some point I had it suddenly working somehow, but for a good half minute before that it seemed impossible to scroll.
